README.md

```diff
@@ -286,71 +270,74 @@ Allowed values are `traditional`, `syslog`, and `modern`. Default to `modern`.
+* `private_key_src`: Local key file path which is copied to the target host. If `private_key` is specified, it is copied to the location. Otherwise, to logging_config_dir.
+* `pki_authmode`: Specifying the authentication mode. `name` or `fingerprint` is accepted. Default to `name`.
+* `permitted_servers`: List of hostnames, IP addresses, fingerprints(sha1), and wildcard DNS domains which will be allowed by the `logging` client to connect and send logs over TLS. Default to `['*.{{ logging_domain }}']`
+
+#### logging_outputs remote_files type
 
 `remote_files` output stores logs to the local files per remote host and program name originated the logs.
 
 Available options:
-- `facility`: Facility in selector; default to `*`.
-- `severity`: Severity in selector; default to `*`.
-- `exclude`: Exclude list used in selector; default to none.
-- `property`: Property in property-based filter; no default
-- `property_op`: Operation in property-based filter; In case of not `!`, put the `property_op` value in quotes; default to `contains`
-- `property_value`: Value in property-based filter; default to `error`
-- `async_writing`: If set to `true`, the files are written asynchronously. Allowed value is `true` or `false`. Default to `false`.
-- `client_count`: Count of client logging system supported this rsyslog server. Default to `10`.
-- `io_buffer_size`: Buffer size used to write output data. Default to `65536` bytes.
-- `remote_log_path`: Full path to store the filtered logs.
+
+* `facility`: Facility in selector; default to `*`.
+* `severity`: Severity in selector; default to `*`.
+* `exclude`: Exclude list used in selector; default to none.
+* `property`: Property in property-based filter; no default
+* `property_op`: Operation in property-based filter; In case of not `!`, put the `property_op` value in quotes; default to `contains`
+* `property_value`: Value in property-based filter; default to `error`
+* `async_writing`: If set to `true`, the files are written asynchronously. Allowed value is `true` or `false`. Default to `false`.
+* `client_count`: Count of client logging system supported this rsyslog server. Default to `10`.
+* `io_buffer_size`: Buffer size used to write output data. Default to `65536` bytes.
+* `remote_log_path`: Full path to store the filtered logs.
                        This is an example to support the per host output log files
                        `/path/to/output/dir/%FROMHOST%/%PROGRAMNAME:::secpath-replace%.log`
-- `remote_sub_path`: Relative path to logging_system_log_dir to store the filtered logs.
+* `remote_sub_path`: Relative path to logging_system_log_dir to store the filtered logs.
 
 **Note:** Selector options and property-based filter options are exclusive. If Property-based filter options are defined, selector options will be ignored.
 
 **Note:** If both `remote_log_path` and `remote_sub_path` are _not_ specified, the remote_file output configured with the following settings.
 
-```
+```java
```

Comment:
And here I guess java is the one?

Reply:
Since it works - yes - but the rsyslog config language is not generally like java